### PR TITLE
Updated music store item spawns

### PR DESCRIPTION
**Pull request by BorkBorkGoesTheCode**
Replaced the "add": violin_golden spawn with a musicstore_showpiece itemgroup derived from all mussto_foo itemgoups
Changed the spawn chances of the items in musicstore_showpiece
Replaced "place_items" with "place_loot"
Added clay jug, bone flute, spoons, spoons and washboard, and MP3 player spawns to musicstore_showpiece (need to add musical use_actions to the jug and washboard)
Buffed golden fiddle spawn chance

### CODE REVIEW

File: data/json/mapgen/musicstore.json [hunk 1]
```diff
@@ -1,4 +1,41 @@
...
+          { "item": "mp3", "prob": 20 },
+          { "item": "saxophone", "prob": 50 },
+          { "item": "trumpet", "prob": 50 },
+          { "item": "tuba", "prob": 25 },
+          { "item": "ukulele", "prob": 50 },
+          { "item": "violin", "prob": 90 },
+          { "item": "violin_golden", "prob": 15 },
+          {
+            "collection": [
+              { "item": "spoon" },
```
Comment (Night-Pryanik) [inline data/json/mapgen/musicstore.json]: Spoons as musical instrument are made of wood, as far as I recall, and spoons in-game are made of steel.

File: data/json/mapgen/musicstore.json [hunk 2]
```diff
@@ -1,4 +1,41 @@
...
+        "distribution": [
+          { "item": "acoustic_guitar", "prob": 100 },
+          { "item": "bagpipes", "prob": 25 },
+          { "item": "banjo", "prob": 60 },
+          { "item": "bone_flute", "prob": 20 },
+          { "item": "clarinet", "prob": 50 },
+          { "item": "flute", "prob": 50 },
+          { "item": "harmonica_holder", "prob": 70 },
+          { "item": "jug_clay", "prob": 20 },
+          { "item": "mp3", "prob": 20 },
```
Comment (Night-Pryanik) [inline data/json/mapgen/musicstore.json]: I feel that mp3-player, not being a musical instrument, is out of theme here.

**Comment by Night-Pryanik** [comment]
Wood saw? https://en.wikipedia.org/wiki/Musical_saw

**Comment by Mecares** [comment]
I would love a low chance to find a violine case with a tommy gun

File: data/json/mapgen/musicstore.json (hunk 1 above)
Comment (BorkBorkGoesTheCode) [reply data/json/mapgen/musicstore.json]: Pun intended?

File: data/json/mapgen/musicstore.json (hunk 2 above)
Comment (BorkBorkGoesTheCode) [reply data/json/mapgen/musicstore.json]: Pun intended?

**Comment by BorkBorkGoesTheCode** [comment]
@Mecares You don't have to use the uncrafting code to ensure multiple items spawn together anymore; the spoon and washboard spawn in this PR does essentially the same thing without as much hassle. What do you think of this:
```

  {
    "id": "case_violin",
    "type": "GENERIC",
    "name": "violin case",
    "name_plural": "violin case",
    "description": "Useful to carry your precious musical instrument around protected from any harm.",
    "weight": 2800,
    "volume": 20,
    "price": 24000,
    "to_hit": -4,
    "bashing": 8,
    "material": [ "plastic", "steel", "leather" ],
    "symbol": "&",
    "color": "dark_gray",
    "covers": [ "ARM_EITHER", "HAND_EITHER" ],
    "coverage": 10,
    "encumbrance": 50,
    "storage": 14,
    "material_thickness": 5,
    "flags": [ "FANCY", "OVERSIZE", "BELTED", "RESTRICT_HANDS", "WATER_FRIENDLY" ]
  },
  {
    "id": "multispawn_example",
    "type": "item_group",
    "//": "Placeholder itemgroup for testing. Add the itemspawns to musicstore_showpiece.",
    "items": [
      {
        "distribution": [
          {
            "collection": [
              { "item": "case_violin" },
              { "item": "golden_fiddle" }
            ], "prob": 15
          },
          {
            "collection": [
              { "item": "case_violin" },
              { "item": "grip" },
              { "item": "thompson_drum" },
              { "item": "tommygun" }
            ], "prob": 15
          }
        ]
      }
    ]
  },

```
I reduced the violin case's storage capacity to 14, reduced to_hit to -4, increased its volume to 20, added plastic for detail, increased bashing to 8, increased encumbrance to 50, and increased material_thickness to 5.

**Comment by Mecares** [comment]
Looks great

File: data/json/items/armor.json [hunk 3]
```diff
@@ -9909,5 +9909,26 @@
     "warmth": 10,
     "material_thickness": 1,
     "flags": [ "VARSIZE" ]
+  },
+  {
+    "id": "case_violin",
+    "type": "GENERIC",
+    "name": "violin case",
+    "name_plural": "violin case",
+    "description": "Useful to carry your precious musical instrument around protected from any harm.  In this case the case contains not an instrument but a Tommygun, a drum magazine and a forward grip, disassemble to get its contents.",
```
Comment (Mecares) [inline data/json/items/armor.json]: You copied the description mentioning the Tommygun and the disassembly hint.

Comment (BorkBorkGoesTheCode) [reply data/json/items/armor.json]: Removed it.

**Comment by BorkBorkGoesTheCode** [comment]
The briefcase SMG would be a better addition to the pizzeria.

**Pull request merged**